socks5.py

```diff
@@ -88,7 +108,7 @@
     resolver = None
 
 
-logging.basicConfig(level=logging.DEBUG)
+logging.basicConfig(level=logging.INFO)
```

Comment:
This should probably be moved closer to the top of the file so it's easy to spot and change. (I should have done that originally)

Reply:
I'll move it to the top in my next commit.